docs/user/azure/account.md

```diff
@@ -0,0 +1,94 @@
+# Create an Azure account
+
+You can sign up for an Azure account here: 
+
+[Azure: Create your Azure free account today][user-create]
+
+This is the welcome screen. You can create a free account or you can purchase services.
+
+![Create account](images/account_Create_your_Azure_free_account.png)
+
+Once you've opted to create an account, you must fill out information about yourself. All fields are required. 
+
+![Step 1](images/account_Sign_up_1.png)
+
+You will need to verify your identity once you've filled out your information. You can either bet called or sent an SMS message.
+
+![Step 2](images/account_Sign_up_2.png)
+
+Once your identity has been verified, you must provide a credit card. If you are creating a free account, this will not be charged.
+
+![Step 3](images/account_Sign_up_3.png)
+
+After everything is filled out, you must agree to the terms and conditions. 
+
+![Step 4](images/account_Sign_up_4.png)
+
+Now that everything is filled out, you can go to the Azure portal.
+
+![Created](images/account_Welcome_to_Microsoft_Azure.png)
+
+This is what the Azure portal looks like. 
+
+![Portal](images/account_Microsoft_Azure.png)
+
+
+# Set up Azure CLI credentials
```

Comment:
also there's already a file for credentials... https://github.com/openshift/installer/blob/1f939d01fc96469c2f2205d9e71d6324b85b8a69/docs/user/azure/credentials.md

Reply:
I've removed the credentials bits from the account doc and updated the credentials page with the extra bits from the account page.